docs/UserGuide.md

```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
-       e.g. Finding with keyword: "corn" will match with "CoRN flour", "coRn FlakeS", "popcorn" but not "`" <br />
-       e.g. Finding with keywords: "corn Chicken beef" will match "beef Chicken", "beef corn", etc. but not "beefcorn"
+    2. Ingredient(s) will be matched as long as a full keyword is in its name.<br />
+       e.g. Finding with keyword: "corn" will match with "cornflour" and "corn flakes" but not "apple core". <br />
 
-### 4.2.6 List Ingredients
+[\[Back to Table of Contents\]](#table-of-contents)
 
-List all the ingredients again after `find ingredient` Operation.
+#### 4.4.6 List Ingredient
+
+Switch to Ingredient [tab](#31-tabs) and list all ingredients.
 
 **Format:**<br />
 `list ingredient`
 
 **Example(s):**<br />
-1. `list ingredient`
+1. Command: `list ingredient`
 <br />Expected Output:<br />
-![listIngredient1.png](images/ingredientCommands/listIngredient1.png)
+![listIngredient1.png](images/ingredientCommands/listIngredient.png)
+
+[\[Back to Table of Contents\]](#table-of-contents)
 
-### 4.2.7 View Ingredients
+#### 4.4.7 View Ingredient
 
-View an ingredient in the [Main Window](#36-mainwindow).
+Open an ingredient in the [Main Window](#36-mainwindow).
 
 **Format:**<br />
 `view ingredient <index>`
 
 **Example(s):**<br />
-1. `view ingredient 3`
-   <br />Expected Output:<br />
-   ![viewCommand1.png](images/ingredientCommands/viewCommand1.png)
+1. Command: `view ingredient 3`
+<br />Expected Output:<br />
+![viewCommand1.png](images/ingredientCommands/viewIngredient.png)
 
 **Additional Information:**<br />
 - An index number is required for the Command. Refer to the indexes displayed for each [Card](#33-cards) in the
   [Side Bar](#32-side-bar).
 
-## 4.3 Recipes
-This sections covers commands related to RecipeBook management. Any command primarily interacting with recipes will
+[\[Back to Table of Contents\]](#table-of-contents)
+  
+### 4.5 Recipes
+This sections covers commands related to Recipe Book management. Any command primarily interacting with recipes will
 be here.
 
-### 4.3.1 Add Recipes
+#### 4.5.1 Add Recipe
 
-Add a recipe to the RecipeBook.
+Add a recipe to the Recipe Book.
 
 **Format:**<br />
-`add recipe -n <name> -i <ingredient> <quantity> [-d <optional description>] [-s <steps>]...`
+`add recipe -n <name> -i <ingredient>... [-d <description>] [-s <steps>]...`
 
 **Example(s):**<br />
-1. `add recipe -n pasta -i tomato 1 -i milk 100ml -i chicken breast 200g -s Chicken thicc -s Thicc chicken`
-    <br />Expected Output:<br />
-    ![addRecipe1.png](images/recipeCommands/addRecipe1.png)
-
-2. `add recipe -n aglio olio -i pasta 200g -d grandmother aglio olio recipe -s aglioli olioli -s aglioli olioli`
-    <br />Expected Output:<br />
-    ![addRecipe2.png](images/recipeCommands/addRecipe2.png)
+1. Minimal command flags used:<br /> 
+Command: `add recipe -n pasta -i tomato 2 -i linguine 50g`
+<br />Expected Output:<br />
+![addRecipe1.png](images/recipeCommands/addRecipe1.png)
```

Comment:
Soz. I'll update the ingredients spacing when there is no quantity provided, can retake the screenshot later after it's merged?

Reply:
You meant when there are no units provided? Okay.